wiegand/api.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/python3
```

Comment:
this file only contains the api, not the main function. this line is not necessary.

Reply:
sorry, forgot to remove that